test/unittest/unittest_nntrainer_layers.cpp

```diff
@@ -853,12 +853,13 @@ TEST_F(nntrainer_BatchNormalizationLayer, forward_backward_training_01_p) {
   sharedConstTensor forward_result;
 
   EXPECT_NO_THROW(forward_result = layer.forwarding(MAKE_SHARED_TENSOR(in)));
-
   matchOutput(*forward_result, "tc_bn_fc_1_goldenBNResultForward.out");
 
-  nntrainer::Tensor backward_result;
-  EXPECT_NO_THROW(backward_result = *layer.backwarding(
-                    MAKE_SHARED_TENSOR(constant(1.0, 3, 1, 1, 12)), 1));
+  nntrainer::Tensor backward_in(layer.getOutputDimension());
+  loadFile("tc_bn_fc_1_goldenBNLayerBackwardDxIn.out", backward_in);
```

Comment:
this test the backwardDx. `AfterUpdate` will be in next unittest?

Reply:
Yes. Do you prefer it to be deleted now?

Reply:
No its fine. I just wanted to know.